packages/ui/src/services/color-config/color-css-helpers-updater.ts

```diff
@@ -0,0 +1,77 @@
+import { watch } from 'vue'
+import { useGlobalConfig } from '../global-config/global-config'
+import { isServer } from '../../utils/ssr-utils'
+import { addOrUpdateStyleElement } from '../dom-functions'
+
+export type HelperConfig = {
+  stylePrefix?: string;
+  stylePostfix?: string;
+  styleProperty?: string;
+  styleValue?: string;
+}
+
+export let helperConfigValues: HelperConfig[] = [
+  {
+    stylePrefix: 'bg',
+    styleProperty: 'background-color',
+  },
+  {
+    stylePrefix: 'text',
+    styleProperty: 'color',
```

Comment:
I think color--primary should use both color and fill.
Maybe change it to styleProperty: ['color', 'fill']

So it should have MaybeArray<string> type.